server/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -254,6 +257,18 @@ public void show_withStringProgramParam_showsByProgramSlug() {
         .contains(routes.ApplicantProgramReviewController.review(program.id).url());
   }
 
+  @Test
+  public void showInfoDisabledProgram() {
```

Comment:
question (maybe blocking): I think it might also be worth adding a browser test so you can test user interaction and take a screenshot of the page. What do you think?

Reply:
Yeah totally! Added a browser test